app/templates/admin/sales.hbs

```diff
@@ -1 +1,18 @@
-{{outlet}}
+<div class="ui grid">
+  <div class="row">
+    <div class="{{if device.isMobile 'sixteen' 'three'}} wide column">
```

Comment:
@niranjan94 I have added tabbed navigation with the above classes. Replacing the above classes with tabbed navigation results in bad UI. I have made the changes.